etl/helpers.py

```diff
@@ -799,6 +707,50 @@ def load_mdim_config(self, filename: Optional[str] = None, path: Optional[str |
     def load_explorer_config(self, filename: Optional[str] = None, path: Optional[str | Path] = None) -> Dict[str, Any]:
         return self.load_config(filename, path)
 
+    def create_dataset(
+        self,
+        tables: Iterable[catalog.Table],
```

Comment:
Is there a reason these args are not passed as **kwargs?

Reply:
Not a strong one, but I noticed that this way you can see the arguments in the code editor. With kwargs I think you don't see them, right?

Reply:
For example, if you are in a data step and you want to use some of the additional arguments of `create_dataset`, but you don't remember the name... Then VSCode will show you all options. But using kwargs (I think?) you would not be able to see them straightaway. Maybe there's a better solution.

Reply:
That's right, kwargs won't expand in autocomplete unless you add some type hinting magic (which looks way worse than propagating arguments). Go with your solution, if it works, we would ideally remove `dest_dir` from old steps anyway and have a single function.